app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Hah I didn't think that was a thing, but I never doubt when Rails adds more automatic things. Seemed weird for something as explicit as scoping to be automated though.

We'll be fine - no need for a vote or to bring back a merged PR.